calendar-app/src/pages/App.tsx

```diff
@@ -0,0 +1,36 @@
+import Header from "./calendar/Header";
+import Content from "./calendar/MainContent";
+import Event from "./calendar/Event";
+import { useState } from "react";
+import { useEventContext } from "../utils/EventContext";
+function App() {
+  const [isCalendarPanel, setCalendarPanel] = useState(true);
+  const [isRightSidePanel, setRightSidePanel] = useState(true);
+  //const [isEventWindow, setEventWindow] = useState(false);
+  const { isEventWindow, setEventWindow } = useEventContext();
+  const { selectedEvent } = useEventContext();
+  return (
+    <div className="App">
+      <Header
+        calendarPanelState={isCalendarPanel}
+        setCalendarPanelDisplay={setCalendarPanel}
+      />
+      <Content
+        calendarPanelState={isCalendarPanel}
+        rightSidePanelState={isRightSidePanel}
+        setRightSidePanelDisplay={setRightSidePanel}
+        eventWindow={isEventWindow}
```

Comment:
naming: `eventWindow` prop suggest something more complex, like an object or event component, but actually is a boolean?